platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatDarkLaf.properties

```diff
@@ -90,6 +95,11 @@ nb.multitabs.project.8.background=rgb(107, 135, 38)
 nb.multitabs.project.9.background=rgb(118, 89, 135)
 nb.multitabs.project.foreground=#f0f0f0
 
+#---- Tab switcher ----
+
+nb.popupswitcher.background=$MenuItem.background
+nb.popupswitcher.selectionForeground=$MenuItem.selectionForeground
+nb.popupswitcher.selectionBackground=$MenuItem.selectionBackground
```

Comment:
> If you want to fix just the tab popup for now, do that, although is there a reason to mix $MenuItem and $List values - just $MenuItem makes sense?

@neilcsmith-net I don't remember, I thought it was because of the "underline menu selection mode" and because the switcher doesn't suppor it - but it works fine -> changed everything to `$MenuItem`